code/datums/supplypacks/hydroponics.dm

```diff
@@ -88,6 +88,12 @@
 	containername = "Chicken crate"
 	access = access_hydroponics
 
+//What a boon!
+/datum/supply_pack/hydro/crabs
+	name = "THREE (3) free crabs"
+	containertype = /mob/living/simple_mob/animal/passive/crab
```

Comment:
This is using the mob as the container? And is also a single crab.

Reply:
Oversight. Will correct later.

Reply:
@lbnesquik any plans to work on this?